### DISCUSSION TITLE
Version 3.0 Y Axis Daten Minimum

### DISCUSSION THREAD

**Discussion by huteel** [Zefau/ioBroker.jarvis#1245] [General]
Wie kann man bei Charts denn die Y-Achse konfigurieren sodass sie nicht bei 0 anfängt, sondern mit etwas Versatz dort wo die Datenpunkte liegen?

Habe versucht die Y Achse mit  {"min":"dataMin"} zu konfigurieren, hat aber nichts gebracht.

**Comment by mcuiobroker** [comment]
```[{"min":"dataMin"}]```
Als array eingetragen?

**Comment by huteel** [comment]
Bewirkt leider auch nichts. Y-Achse startet immer noch bei 0. Hoffe liegt nicht daran dass ich nicht die Pro Version habe? Finde es eh komisch dass essentielle Funktionen hinter einer Paywall verschwinden.

**Reply by mcuiobroker** [reply]
Alle Einstellungen bzgl Achsen im HistoryGraph sind Bestandteil der PRO, soweit ich es weiß.

**Comment by Zefau** [comment]
Ja, ist Pro. Siehe auch https://github.com/Zefau/ioBroker.jarvis/discussions/891. Und als essentiell würde ich das nicht sehen.
